packages/plugins/src/cli/propose/index.ts

```diff
@@ -189,16 +190,18 @@ export const buildParsedConfigArray: BuildParsedConfigArray = async (
     // transactions broadcasted in the user's script for this network. We return an empty array in
     // this case.
     const actionInputs = collectionDryRun
-      ? convertFoundryDryRunToActionInputs(
-          deploymentInfo,
-          collectionDryRun,
-          collectionDryRunPath
-        )
+      ? convertFoundryDryRunToActionInputs(deploymentInfo, collectionDryRun)
```

Comment:
We now use the dry run path as an input to `makeParsedConfig`. (Likewise for the `deploy` function)